File: algorithm_leetcode/src/test/scala/com/huldar/test/scala/day02/PalindromeTest.scala
package com.huldar.test.scala.day02

import com.huldar.scala.day02.Palindrome
import org.scalatest.FunSuite

/**
  *
  * TODO
  *
  * @author huldar
  * @date 2019/6/10 15:54
  *
  */
class PalindromeTest extends FunSuite {

  test("testIsPalindrome") {
    assertResult(true)(new Palindrome().isPalindrome(121))
    assertResult(false)(new Palindrome().isPalindrome(-1))
    assertResult(false)(new Palindrome().isPalindrome(12))
  }

}

File: algorithm_leetcode/src/test/java/com/huldar/test/java/day02/PalindromeTest.java
package com.huldar.test.java.day02;

import com.huldar.java.day02.Palindrome;
import org.junit.jupiter.api.Test;

import static org.junit.Assert.*;

/**
 * 测试类
 *
 * @author huldar
 * @date 2019/6/5 10:22
 */
public class PalindromeTest {
    @Test
    void isPalindromeTest() {
        assertEquals(true, Palindrome.isPalindrome(121));
        assertEquals(false, Palindrome.isPalindrome(12));
        assertEquals(false, Palindrome.isPalindrome(-121));
        assertEquals(true, Palindrome.isPalindrome(11));

    }
}
